standard-covoiturage_openapi.yaml

```diff
@@ -116,6 +128,301 @@ paths:
+                            $ref: '#/components/schemas/Schedule'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        429:
+          $ref: '#/components/responses/TooManyRequests'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+
+  /messages:
+    post:
+      tags:
+      - Interact
+      summary: Send a message to the owner of a retrieved journey.
+      description: Route used to allow a user to connect back to the owner of a retrieved journey through a texte message.
+      operationId: postConnections
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/ConnectionMessage'
+              type: object
+              required:
+                - message
+                - recipientCarpoolerType
+              properties:
+                recipientCarpoolerType:
+                  type: string
+                  enum:
+                    - driver
+                    - passenger
+                  default: driver
+                  description: Defines if the recipient of this message is either the driver or the passenger.
+      responses:
+        201:
+          description: Successful operation.
+        404:
+          description: The targeted journey or user no more exists.
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        429:
+          $ref: '#/components/responses/TooManyRequests'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+
+  /bookings:
+    post:
+      tags:
+      - Interact
+      summary: Create a punctual outward Booking request.
+      description: Route used to synchronize a Booking request initiated by a platform to the second platform involved in the shared punctual outward journey. While posting a new Booking, its status shall always be set first as `status=WAITING_CONFIRMATION`. The  Booking MUST be created using the relevant `passengerJourneyId` or `driverJourneyId` attributes according to the operator role handling the request (the operator can recognize its own role thanks to the `driverOperator` and `passengerOperator` attributes).
+        _Reminder:_ In case of booking without deeplink, the sender platform MUST also store the Booking object, and be ready to receive modifications of it through the PATCH /bookings API endpoint.
+      operationId: postBookings
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Booking'
+                  
+      responses:
+        201:
+          description: Successful operation. A new Booking has been created.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Booking'
```

Comment:
As we discussed, the price in the search is just an estimation (that does not know about user information). 
At this point, is it a real or estimated price ? What happens if the price differs from the first estimation ?

Reply:
The price returned during the search is not an estimation, but the price expected by the carsharing operator from the MaaS. However, the end user may pay a different price.